release/goreleaser.opm.Dockerfile

```diff
@@ -2,7 +2,11 @@
 #   build opm images. See the configurations in .goreleaser.yaml
 #   and .github/workflows/release.yaml.
 
-FROM --platform=$BUILDPLATFORM ghcr.io/grpc-ecosystem/grpc-health-probe:v0.4.4 as grpc_health_probe
+FROM ubuntu as grpc_health_probe
+RUN apt update && \
+  apt install -y curl && \
+  curl -o /grpc_health_probe \
```

Comment:
```suggestion
  curl --silent --show-error --location --output /grpc_health_probe \
```

Reply:
Working on this line too, looks like curl has some issues on arm64 =( probably we will have to switch to wget

Reply:
what about using -SsL ? what do you think?

Reply:
In scripts, I generally try to remember to use long form options to make it easier to grok later.

Reply:
I'm going to push new version with the long options